src/Nethermind/Nethermind.Blockchain.Test/Validators/TxValidatorTests.cs

```diff
@@ -215,5 +215,27 @@ public bool MaxFeePerGas_is_required_to_be_greater_than_MaxPriorityFeePerGas(TxT
             TxValidator txValidator = new(TestChainIds.ChainId);
             return txValidator.IsWellFormed(tx, London.Instance);
         }
+
+        [TestCase(TxType.EIP1559, false, ExpectedResult = false)]
```

Comment:
It's false because of `transaction.MaxFeePerGas` < `transaction.MaxPriorityFeePerGas` in `Validate1559GasFields`, looks like in `TransactionBuilder` default value of `MaxFeePerGas` is 0 and `MaxPriorityFeePerGas` is 1. Maybe we need to change it? It's totally not related to this PR so you can fix test by just adding `.WithMaxFeePerGas(1)` when building tx.

Reply:
Or `tx.DecodedMaxFeePerGas = tx.MaxPriorityFeePerGas;` just after building test tx